pkg/defaults/defaults.go

```diff
@@ -14,9 +14,17 @@
 
 package defaults
 
-import "github.com/sirupsen/logrus"
+import (
```

Comment:
should we keep a `pkg/default/defaults.go` or `pkg/default/daemon.go` for the daemon related defaults, and a `node.go` for the node's package related defaults and so on.

What about k8s specifics defaults, should they be in `defaults.go` as well?

Reply:
Yes, I'm fine with that. If we move all defaults, we can split them into files.